src/propertiesdialog.cpp

```diff
@@ -469,9 +469,9 @@ void PropertiesDialog::setupShortcuts()
 
 void PropertiesDialog::bookmarksButton_clicked()
 {
-    QFileDialog dia(this, tr("Open or create bookmarks file"));
+    QFileDialog dia(this, tr("Select bookmarks file"));
     dia.setOption(QFileDialog::DontConfirmOverwrite, true);
-    dia.setFileMode(QFileDialog::AnyFile);
+    dia.setNameFilter(tr("*.xml files (*.xml)"));
```

Comment:
The above two lines should be:
```c++
    dia.setFileMode(QFileDialog::ExistingFile);
    dia.setNameFilter(tr("XML files (*.xml)"));
```
But please read my next comment before changing them.